### DISCUSSION TITLE
Will we ever get Material Design 3 to Material Design In XAML?

### DISCUSSION THREAD

**Discussion by olifurz** [MaterialDesignInXAML/MaterialDesignInXamlToolkit#2684] [Q&A] [answered]
_see title_

**Comment by ElieTaillard** [comment, marked as answer]
Yes it's in Development

https://github.com/MaterialDesignInXAML/MaterialDesignInXamlToolkit#-source-code
> The last one is a Material Design 3 project (In Development).

Here is a Demo project with Material Design 3:
https://github.com/MaterialDesignInXAML/MaterialDesignInXamlToolkit/tree/master/MaterialDesign3.Demo.Wpf

You can check the Roadmap here:
https://github.com/MaterialDesignInXAML/MaterialDesignInXamlToolkit/issues/2475
